.mailmap

```diff
@@ -548,3 +550,14 @@ Muhammed Abdul Quadir Owais <quadirowais200@gmail.com> MaqOwais <quadirowais200@
 Vaibhav Bhat <vaibhav.bhat2097@gmail.com> VBhat97 <vaibhav.bhat2097@gmail.com>
 Bhaskar Joshi <bhaskar.joshi@research.iiit.ac.in> BhaskarJoshi-01 <bhaskar.joshi@research.iiit.ac.in>
 Miguel Torres Costa <miguelptcosta1995@gmail.com>
+Elisha Hollander <just4now666666@gmail.com> donno2048 <just4now666666@gmail.com>
+Mamidi Ratna Praneeth <praneethratna@gmail.com> praneeth <praneethratna@gmail.com>
+Zach Carmichael <20629897+craymichael@users.noreply.github.com> Zach <20629897+craymichael@users.noreply.github.com>
+Anup Parikh <parikhanupk@gmail.com> Anup Parikh <84572003+parikhanupk@users.noreply.github.com>
+Harshit Yadav <harshityadav2k@gmail.com> Harshit Yadav <45384915+hyadav2k@users.noreply.github.com>
+Megan Ly <megan@artcompiler.com> Megan Ly <meganly@MacBook-Pro.local>
+Robert Pollak <robert.pollak@posteo.net> Robert Pollak <robert.pollak@jku.at>
+gum3ng <gum3ng@protonmail.com> gum3ng <78410304+gum3ng@users.noreply.github.com>
+mostlyaman <amansharma110603@gmail.com> mostlyaman <85072876+mostlyaman@users.noreply.github.com>
+robinechuca <raisin@ecomail.fr> robinechuca <61911191+robinechuca@users.noreply.github.com>
+Huangduirong <huangduirong@huawei.com> Huangxiaodui <hdrong.42@163.com>
```

Comment:
I added this name/email after an email exchange with the recorded email address.